tests/unit/NameRequestInfo.spec.ts

```diff
@@ -232,14 +232,16 @@ describe('Numbered Amalgamation Info component', () => {
     wrapper.destroy()
   })
 
-  it('renders numbered company info', () => {
-    expect(wrapper.vm.$el.querySelector('#numbered-amalgamation-info').textContent)
+  // *** TODO: fix this
```

Comment:
I will fix these in a future PR.

Meanwhile, unless bugs are found, THIS PR is ready to go.

Reply:
We can add the checkbox to fix these tests in the part 4 ticket if you want Sev: https://github.com/bcgov/entity/issues/18641

Reply:
Good idea. I've added a note to the new ticket.